webhook/resources/backingimage/mutator.go

```diff
@@ -102,12 +103,28 @@ func (b *backingImageMutator) Create(request *admission.Request, newObj runtime.
 		patchOps = append(patchOps, `{"op": "replace", "path": "/spec/disks", "value": {}}`)
```

Comment:
I am thinking, should Longhorn reject the requests if users want to update the deprecated fields to a non-empty value?

Reply:
users can only update the `spec.disks` and `spec.diskFileSpecMap` through command line `kubectl edit ...`
Besides, `spec.disks` is not used in any code.
I think it is not a must but only better have.